datasetinsights/datasets/groceries_real.py

```diff
@@ -324,17 +324,16 @@ def download(self):
         """Download dataset from GCS
         """
         cloud_path = f"gs://{const.GCS_BUCKET}/{self.GCS_PATH}"
+        client = GCSClient()
```

Comment:
Please remove download code in GroceriesReal. `Dataset` class will not download files going forward, and GroceriesReal is a public dataset that can be downloaded through http directly.

Reply:
You mean GoogleGroceriesReal? GroceriesReal does not have any download code.